dataprofiler/profilers/profile_builder.py

```diff
@@ -1285,6 +1289,9 @@ def _get_row_has_null_ratio(self):
     def _get_duplicate_row_count(self):
         return self.total_samples - len(self.hashed_row_dict)
 
+    def _get_correlation(self):
+        return self.correlation_matrix
+
```

Comment:
what's the intent here? If get correlation just returns an existing property?

An intent I can see for having `_get_correlation` would be to separate `_get_correlation` from merging the correlation where `_get_correlation` would only calculate the correlation for the new series before merging or if this is the first update of the correlation would provide the correlation for the profile.

Such workflow would allow the `_get_correlation` and merging of correlations to be timed separately without overlap of function timing

Reply:
the intent is to get top correlation pairs. Let's say, we have many columns, and we only want to show the correlations of the top 10.

Reply:
Currently, this just returns the whole matrix though right? Based on the other functions this might be a bit confusing as the other `_get_*` appear to do a calculation of the statistic referenced. Also, I think the separation as mentioned above is important so we can start adding time tracking to the top level profilers as well.

Reply:
I'm not completely follow what you mentioned above, but my understanding is to separate calculate corr from scratch vs update corr from an existing one? It maybe good, but currently we don't have that for any other stat calculation. Should be a separate PR for all of them?

About the timing, do we have the timing update for get_row_statistics? If yes, we can just do the same as they are at the same level?